app/src/main/java/com/tyron/code/ui/iconmanager/IconManagerActivity.java

```diff
@@ -0,0 +1,432 @@
+import android.net.Uri;
+import android.text.TextUtils;
+import android.provider.DocumentsContract;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.LinearLayout;
+import android.widget.ImageView;
+import android.widget.Toast;
+import androidx.appcompat.app.AlertDialog;
+import androidx.appcompat.app.AppCompatActivity;
+import android.os.Bundle;
+import androidx.appcompat.widget.Toolbar;
+import androidx.recyclerview.widget.GridLayoutManager;
+import androidx.swiperefreshlayout.widget.SwipeRefreshLayout;
+import com.tyron.code.ui.iconmanager.loader.LoaderDialog;
+import com.tyron.code.ui.iconmanager.util.FileUtil;
+import android.widget.Button;
+import androidx.recyclerview.widget.RecyclerView;
+import com.tyron.code.ui.iconmanager.adapter.IconAdapter;
+import com.google.android.material.textfield.TextInputEditText;
+import com.google.android.material.textfield.TextInputLayout;
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
+import java.io.BufferedOutputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileOutputStream;
+import java.net.URLDecoder;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+import com.tyron.builder.project.Project;
+import com.tyron.code.R;
+import com.tyron.code.ui.project.ProjectManager;
+import org.apache.commons.io.FileUtils;
+import com.tyron.common.util.AndroidUtilities;
+import com.google.android.material.dialog.MaterialAlertDialogBuilder;
+
+public class IconManagerActivity extends AppCompatActivity {
+	
+	
+	//mainTask: codeCleanUp :)
+
+	FloatingActionButton fab;
+	Toolbar toolbar;
+	RecyclerView recyclerview1;
+	public static String desPath, resPath, name, icon, vector, unziped_vector, f, project_path, from, from2, samplePath;
+	int n = 0;
+        //ArrayList
+	ArrayList<String> list = new ArrayList<>();
+	ArrayList<String> scanner1 = new ArrayList<>();
+	ArrayList<String> scanner2 = new ArrayList<>();
+        public static ArrayList<String> scanner3 = new ArrayList<>();
+        public static ArrayList<String> scanner4 = new ArrayList<>();
+        //
+	SwipeRefreshLayout swipeRefreshLayout;
+	Project mProject;
+
+	@Override
+	protected void onCreate(Bundle savedInstanceState) {
+		super.onCreate(savedInstanceState);
+		setContentView(R.layout.activity_main);
+		fab = findViewById(R.id.fab);
+		toolbar = findViewById(R.id.toolbar);
+		recyclerview1 = findViewById(R.id.recyclerview1);
+		swipeRefreshLayout = findViewById(R.id.s1);
+
+		//Strings
+		desPath = FileUtil.getPackageDir(this) + "/material-icons-pack/materialiconsoutlined/preview-packs/";
+		String mPath = FileUtil.getPackageDir(this) + "/material-icons-pack/";
```

Comment:
Wrong naming of variables, see https://source.android.com/setup/contribute/code-style#follow-field-naming-conventions